pkg/okteto/client.go

```diff
@@ -245,7 +251,7 @@ func InDevContainer() bool {
 	return false
 }
 
-func query(ctx context.Context, query interface{}, variables map[string]interface{}, client *graphql.Client) error {
+func query(ctx context.Context, query interface{}, variables map[string]interface{}, client graphqlClientInterface) error {
 	err := client.Query(ctx, query, variables)
 	if err != nil {
 		if isAPITransientErr(err) {
```

Comment:
can we test this case? when err is transient

Reply:
I plan to test that scenario when we test the `client.go` file, but I prefer to go file by file testing to avoid confussion while reviewing